src/qml/Mysthea/Pages/DetailDelegate.qml

```diff
@@ -20,51 +26,132 @@ Pane {
         ColumnLayout {
             id: _layout
             width: parent.width
-
-            spacing: 8
-
-            Label {
-                font.pixelSize: 24
-                text: root.code
-
-                Layout.alignment: Qt.AlignHCenter
-            }
-
-            RowLayout {
-
+            spacing: 24
+            ColumnLayout {
+                width: parent.width
                 spacing: 8
+                RowLayout {
+                    width: parent.width
+                    spacing: 12
+                    Label {
+                        text: root.code
+                        font.pixelSize: 32
+                        font.letterSpacing: 0.5
+                        font.weight: Font.Bold
+                        color: {
+                            if (root.command === "Tactic") {
+                                return Palette.flamingo
+                            } else if (root.command === "Objective") {
+                                return Palette.goldenFizz
+                            } else if (root.command === "Accessory") {
+                                return Palette.apple
+                            } else if (root.command === "Upgrade") {
+                                return Palette.cerulean
+                            } else {
+                                return Palette.grayNurse
+                            }
+                        }
+                    }
 
-                Image {
-                    fillMode: Image.PreserveAspectFit
-                    source: "qrc:/images/cards/" + image
+                    Label {
+                        text: "•"
+                        font.pixelSize: 24
+                        color: Palette.grayNurse
+                    }
 
-                    Layout.fillWidth: true
+                    Label {
+                        font.pixelSize: 32
+                        font.letterSpacing: 0.5
+
+                        text: {
+                            if (root.type == "Hero") {
```

Comment:
This check could be better. Only Command cards have a command property, we could check on that.